components/script/dom/htmlscriptelement.rs

```diff
@@ -104,7 +103,7 @@ impl ScriptSource for ScriptOrigin {
     script_kind: ExternalScriptKind,
     final_url: ServoUrl,
     url: ServoUrl,
-    task_source: DOMManipulationTaskSource,
+    task_source: NewTaskSource,
```

Comment:
Should this be just `TaskSource`?

Reply:
Thanks for catching this! Yeah, this should be `TaskSource`. This is one reason it isn't great to have commented out code.